app/services/intake_progress_calculator.rb

```diff
@@ -22,9 +25,9 @@ def self.get_progress(controller, intake)
     current_index = steps_for_intake.index(controller)
     completed_steps = current_index + 1 # Everything that the client has done
 
-    index_of_possible_steps = POSSIBLE_STEPS.index(controller)
-    possible_remaining_steps = POSSIBLE_STEPS[index_of_possible_steps + 1..-1].length # Everything the client could potentially do
+    index_of_intake_steps = steps_for_intake.index(controller)
+    remaining_steps = steps_for_intake[index_of_intake_steps + 1..-1].length # Everything the client could potentially do
```

Comment:
this is a change in behavior i want to make sure is reviewed. this makes it so steps that we know won't be included based on the data we've collected won't be considered in the progress calculation. clearly it was intentional to include them in the past, but i can't understand why because the progress meter was less accurate. 

am i missing something?

Reply:
also, the reason to do this is that there are a ton of Document controllers in that flow so the status bar looks like you're way farther away from done than you actually are without this change.

but if this change is bad for some reason, i think the alternative would be to only add document controllers where the document type is needed_if_relevant? to the list of POSSIBLE_STEPS because there aren't that many of those.